### PR TITLE
Add unbind method

### PR DESCRIPTION
**Pull request by viartemev**
Issue: https://github.com/vert-x3/vertx-rabbitmq-client/issues/25

### CODE REVIEW

**Comment by viartemev** [comment]
I didn't find test for `bind` method. From my experience it should be testet with running rabbitmq instance. Do you have any plans to add this type of tests?

**Comment by vietj** [comment]
would you mind to add test for bind/unbind ? this module needs contribution and it would be great if you could help us

**Comment by viartemev** [comment]
@vietj could you review it?

**Comment by vietj** [comment]
can you avoid unnecessary code reformat ?

**Comment by viartemev** [comment]
@vietj done. Maybe it makes sense to add style checker and guide for the project?

**Comment by viartemev** [comment]
@vietj ping

File: pom.xml [hunk 1]
```diff
@@ -94,6 +94,25 @@
...
     </dependency>
 
+    <dependency>
+      <groupId>com.rabbitmq</groupId>
+      <artifactId>http-client</artifactId>
+      <version>3.1.1.RELEASE</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
```
Comment (vietj) [inline pom.xml]: why is Spring necessary for this test ?

Comment (viartemev) [reply pom.xml]: @vietj I use HOP client for RabbitMQ: https://github.com/rabbitmq/hop#maven

**Comment by vietj** [comment]
it looks great but can we use vertx HTTP client instead of Spring ?

File: pom.xml (hunk 1 above)
Comment (vietj) [reply pom.xml]: can we avoid adding this dependency and use vertx client instead ?

Comment (vietj) [reply pom.xml]: Ping

**Pull request closed without merging**